pkg/security/auth.go

```diff
@@ -214,13 +215,25 @@ func GetCertificateUserScope(
 	for _, uri := range peerCert.URIs {
 		uriString := uri.String()
 		if URISANHasCRDBPrefix(uriString) {
```

Comment:
This is existing code, but I find this code a little bit surprising.  Namely, we already have a parsed URI but do all of this string comparison on the full string while we could be looking at the already parsed `uri.Scheme`, `uri.Host`, `uri.Path`. Doesn't have to happen here, but I wonder if using the parsed URI might make this code a bit cleaner.